pymysql/connections.py

```diff
@@ -604,6 +606,28 @@ def __init__(self, host=None, user=None, password="",
         if self._local_infile:
             client_flag |= CLIENT.LOCAL_FILES
 
+        if login_path:
+            if login_path is True:
+                login_path = 'client'
+
+            from .login_path import open_login_path_file
+            login_path_file = open_login_path_file()
```

Comment:
I don't like this design.
Please move parsing code to login_path module too.

```
opts = login_path.read()
# opts is dict if successfully read option file.
# opts is None if cryptography is not available, or failed to open/read/decrypt mylogin.cnf
```

With this API, people can test login_path file easily, without creating real connection.